servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -642,6 +672,22 @@ public interface DelayedRetry {
         Duration delay();
     }
 
+    /**
+     * Callbacks invoked on a retry attempt.
+     */
+    @FunctionalInterface
+    public interface RetryCallbacks {
+
+        /**
+         * Called before a retry is performed.
```

Comment:
Wdyt about clarifying the semantics a bit more - i..e `Called after a retry decision has been made, but before the retry is performed`. To make sure that the user does not try to influence the retry decision as part of that callback.